opal/core/search/extract.py

```diff
@@ -252,6 +252,42 @@ def __init__(self, model, episode_queryset, user, fields=None):
         )
 
 
+def generate_files(root_dir, episodes, user):
+    """ Generate the files and return a tuple of absolute_file_name, file_name
```

Comment:
do you say full extract because this will be in contrast to having inline extracts?